fontFeatures/feaLib/Substitution.py

```diff
@@ -40,37 +40,86 @@ def paired_ligature(self):
             replacements.append(j)
     rhs = zip(*replacements)
     for l, r in zip(lhs,rhs):
-        b.statements.append(
-            feaast.LigatureSubstStatement(
+        stmt = feaast.LigatureSubstStatement(
             [glyphref(x) for x in self.precontext],
             [glyphref([x]) for x in l],
             [glyphref(x) for x in self.postcontext],
             glyphref([r[0]]),
             False,
         )
+        b.statements.append(stmt)
+    return b
+
+# Expand multiple substitutions, such that:
+#   * Substitute [a b] -> before_tail [a.2 b.2] tail;
+# Becomes in FEA:
+#   * sub a by before_tail a.2 tail;
+#   * sub b by before_tail b.2 tail;
+def paired_mult(self):
+    b = feaast.Block()
+
+    if len(self.input_lengths) != 1:
```

Comment:
I'm going to rework this `self.input_lengths` stuff; I don't like the fact that it relies on a side-effect from another function, whose stated job is to test something different.

Reply:
I don't mind doing that for you.

Reply:
Done: https://github.com/simoncozens/fontFeatures/pull/14/commits/487dbd1201ef199351f424f842385621abafc0b2